### PR TITLE
Test python 3.8 and Django 3.0

### PR DESCRIPTION
**Pull request by hamstap85**
Aligns the doc specs to match tested versions

We're trying to upgrade to Django 3.0 as soon as possible, so I'm going through all our dependencies and helping ensure they work on the latest version of everything.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 
 def listify(filename):
-    return [line for line in open(filename, 'r').read().split('\n') if line]
+    with open(filename, 'r') as f:
```
Comment (romgar) [inline setup.py:5]: Why this change?

File: tests/test_memory_leak.py [hunk 2]
```diff
@@ -13,4 +13,5 @@ def test_rss_usage():
     for _ in range(1000):
         DirtyMixinModel()
     rss_2 = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss
-    assert rss_2 == rss_1, 'There is a memory leak!'
+    diff_percentage = 100 * abs(rss_2 - rss_1) / rss_1
+    assert diff_percentage <= 1, 'There is a memory leak!'
```
Comment (romgar) [inline tests/test_memory_leak.py:17]: Why this change?

File: setup.py (hunk 1 above)
Comment (hamstap85) [reply setup.py:5]: So that the file object doesn't remain open

File: tests/test_memory_leak.py (hunk 2 above)
Comment (hamstap85) [reply tests/test_memory_leak.py:17]: I'm not entirely convinced that an exact match is the only indication of no memory leak, because of garbage collection. Specifically because it was starting to fail all the time with minuscule amounts.

**Comment by LincolnPuzey** [comment]
Thanks @hamstap85 This is now in the develop branch. I will close this PR.
